src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/MD5.cs

```diff
@@ -14,7 +14,10 @@ namespace System.Security.Cryptography
 
     public abstract class MD5 : HashAlgorithm
     {
-        protected MD5() { }
+        protected MD5()
```

Comment:
Nitpick: Maybe add a new constructor to `System.Security.Cryptography.HashAlgorithm` to take a `HashSizeValue` and implement these as `protected MD5() : base(hashSize: 128)`?

Reply:
That'd be new public surface area.  We have a dedicated process for adding surface area like that, requiring discussion / strict review / etc.